src/app/public/modules/checkbox/checkbox.component.ts

```diff
@@ -119,11 +99,44 @@ export class SkyCheckboxComponent implements ControlValueAccessor, Validator {
     return this._checked;
   }
 
-  private control: AbstractControl;
+  /**
+   * Indicates if the checkbox must be checked to be valid. This property accepts a boolean values.
+   */
+  @Input()
+  public required: boolean;
+
   private isFirstChange = true;
   private _checkboxType: string;
   private _checked: boolean = false;
 
+  constructor(
+    @Self() @Optional() private ngControl: NgControl
```

Comment:
Yes, for checkboxes without an `ngModel` (template-driven) or inside a form group (reactive). Without the `@Optional()`, it will through a `No provider for NgControl` error in those situations.